fusionfusion/annotationFunction.py

```diff
@@ -271,4 +271,28 @@ def merge_fusion_result(input_dir, output_file_path):
     hOUT.close()
 
     subprocess.check_call(["rm", "-rf", output_file_path + ".unsorted.tmp"])
+
+    trace_files = list(filter(os.path.exists, [
+        input_dir + "/star.chimeric.trace.txt",
+        # input_dir + "/ms2.chimeric.trace.txt",
+        # input_dir + "/th2.chimeric.trace.txt"
+    ]))
+    if trace_files:
+        traced_file_path = re.sub('(?:\.txt)?$', '.traced.txt', output_file_path, count=1)
```

Comment:
https://www.flake8rules.com/rules/W605.html
```suggestion
        traced_file_path = re.sub(r'(?:\.txt)?$', '.traced.txt', output_file_path, count=1)
```